src/server/object_services/object_server.js

```diff
@@ -698,7 +698,7 @@ async function read_object_mapping(req) {
     const obj = await find_object_md(req);
 
     // Check if the requesting account is authorized to read the object
-    if (!await req.has_s3_bucket_permission(req.bucket, 's3:GetObject', '/' + obj.key)) {
+    if (!await req.has_s3_bucket_permission(req.bucket, 's3:GetObject', '/' + obj.key, null)) {
```

Comment:
I think that sometimes you send `undefined` and sometimes `null` (if so it is confusing).

Reply:
Yeh, I should be consistent.